### PR TITLE
Deploy staging to production (week 31-32)

### PR DESCRIPTION
**Pull request by Entkenntnis**
Contains some work-in-progress stuff.

I18n and Authenification needs to be enabled in the cloudflare worker. Small breaking change: redirections are not handled by the frontend anymore.

Deploy after https://github.com/serlo/serlo.org-cloudflare-worker/pull/37

### CODE REVIEW

File: src/fetcher/get-initial-props.ts
Comment (inyono) [inline src/fetcher/get-initial-props.ts]: Not happy about that. Can we cover that in the fetcher itself (Namely checking if uuid / whatever is null)? The actual error message might change depending on which node version Vercel is running on.

**Review by inyono — CHANGES_REQUESTED** [review]

File: src/fetcher/get-initial-props.ts
Comment (Entkenntnis) [reply src/fetcher/get-initial-props.ts]: The request throws an exception with this message: 
grafik

Can the API avoid throwing this error and just pass uuid as null instead? Otherwise the frontend is not much able to tell what happened.

Comment (inyono) [reply src/fetcher/get-initial-props.ts]: Ah, didn't know that this happens at API level. My bad, gonna look into this.

Comment (Entkenntnis) [reply src/fetcher/get-initial-props.ts]: No problem, thanks for investigating this.

**Comment by inyono** [comment]
Won't merge this today since I don't know if it's in a deployable state (especially considering i18n / auth).

**Review by inyono — CHANGES_REQUESTED** [review]
Sorry, merging still blocked by Cloudflare Worker, see https://github.com/serlo/serlo.org-cloudflare-worker/issues/40

**Review by inyono — APPROVED** [review]
Okay, Cloudflare Worker is finally ready.

**Pull request merged**
